### ISSUE TITLE
Unknown Numbers

### ISSUE THREAD

**Issue by mrmakeit** [mrmakeit/PrototypeHARP-Research#5]
What do the numbers 588872943124282030399478301168915802302423130847520232836494683907672278768724884179218375754768 mean?

**Comment by mrmakeit** [comment]
Available Decoder suggests `fjtpexbtdmpzdkpmfbdxwmhuzbfelppmgovzxixjoauewwup`, but that doesn't seem to work anywhere just yet.

**Comment by yotann** [comment]
Have you ever used a decoder ring in real life? I think I got one in a cereal box once.

**Comment by mrmakeit** [comment]
Ya, but what I was given was more of a decoder wheel, and just matches a-z to 1-100 with the values repeating after 26.  I will look at uploading the file I have that returned the output above.

**Comment by yotann** [comment]
I mean, there's more to it than that. A real decoder ring/wheel has moving parts, and in this case you have to move them in order to solve the puzzle.

**Comment by mrmakeit** [comment]
Fair point.  Since it is an image on a pdf, I've gone through and added a javascript tool to try and simulate the decoders results in various ways.  No results yet.

**Comment by mctom987** [comment]
The board's RC1 and RC2 match up a little too closely to the decoder wheel's 1st and 2nd wheels. Haven't been able to decode anything with this, however.

**Comment by yotann** [comment]
You're almost there; you just have to find the right key for the decoder wheel. After you decode the message you'll probably want to get your camera ready.

**Comment by mrmakeit** [comment]
@yotann Thanks for the hint.  Unfortunately I've hit a brick wall on this key.  I'm guessing I've missed something obvious.

**Comment by yotann** [comment]
The key is a four-letter word from the same message as the wheel. You have to rotate the boxed numbers (01, 27, 53, 79) to line up with the corresponding letters of the key, then use the wheel to decode the message. Watch out for off-by-one errors!

**Comment by mrmakeit** [comment]
Found the bug with my tools, but still end up with garbage data.  I'm going to try poking these values on the board, but not expecting much.
`HOPE = qfelqqnmpylkpdaiqupqpfaglurawlayzzokieqfawnahhga`
`HARP = oucaeqbmdmaiddyxoudqpfauzufpuaymzxoigtquolnpffuy`

**Comment by yotann** [comment]
The key is `HOPE` and the decoded message is `unitseparatorrequiredtoinitiateandcommencebillie`. (That's one of the lines in your `options.txt`, so you were definitely on the right track when you made that file.)

**Comment by putnam** [comment]
I'm revisiting my puzzle after solving it years ago, and somehow cannot figure out what to do next after decoding the message. Surprisingly very little out there about this puzzle besides this repo. Sorry for the necro, but do any of you remember enough to give me a hint to move on? :)

**Comment by xrm** [comment]
@Faeranne Sorry for grave-digging this further, but ...

@putnam Just searched through my notes and found this:
1) I powered up the board, put in the resistor, corrected the address for booting the board.
2) Hit CTRL+D, got the numbers, decoded them with HOPE and the mail.
3) *Almost* broke the board in two halves but realized that Unit Seperation means something different ...
4) Did the separation, Billie came up.
(I'm not sure about the order of the next few things, it was late when I did that)
5) Cut the antenna to take the board offline.
6) Measured the voltages, helped Billie to debug itself.
7) Ordered a BioS (3 $ for the chip, 22 $ for shipping ... ouch), put it in, Billie booted completely, gave the password for the Oktoberfest
8) Decrypted Oktoberfest, read through the documents, viewed the videos, understood that the filenames are base64-encoded; decoded them.
9) Went back to Billie who got mad at me because I was gone for so long, said he'll hack everything around.

Does that help?


... also, if anyone remembers how to get the revision code for the website, I'd be more than grateful ...

**Comment by putnam** [comment]
Very helpful -- I remember most of this as well, but just can't remember how to do step 4 (the separation). I'm sure I'm missing something embarrassing. Do you remember anything about that step that you can provide as a hint?

I also had notes like yours, as in, a full write-up from beginning to end, but I deleted it after thinking I'd never see the puzzle again...

**Comment by putnam** [comment]
I also seem to remember the BIOS was not strictly necessary. It was something you could bypass by downloading a file somewhere instead. (I did buy one though)

**Comment by xrm** [comment]
@putnam Sure! It's probably better to look for a "unit separator" than for an actual separation.

Let me know if that helps (or not), I can also be more specific.

Cheers! ^_

**Comment by putnam** [comment]
Could you be a little more specific? It may sound dumb right now, but I’ve
tried inputting various kinds of separators like commas, symbols, etc. to
no avail.

This time I’ll have to do a full write up with spoilered clues for future
puzzlers.

**Comment by xrm** [comment]
Of course!

It's not a, but *the* unit separator.

(See "unit separator ascii" in a search engine of your choice). :-)

Better?

Cheers! ^_

**Comment by putnam** [comment]
Told you it’d be embarrassing! Thanks!

**Comment by putnam** [comment]
Having finished the puzzle again, I forgot about the incomplete feeling of everything. There are multiple puzzles that look abandoned or pieces never used, like the whole "file scrambling" document that goes to nothing, the "look in the mirror" easter egg, etc.

They even provided hex editors and instructions on how to use them, something that is only useful if they actually implemented the "encryption" puzzle. Would have been cool, and I'm sure some people wasted a lot of time thinking they needed to "decrypt" certain image/video files that served no other purpose on the SD card.

**Comment by xrm** [comment]
I was about to point at that part of the puzzle just before I saw your edit ... :-)

When I solved it the first time, I actually tried both ways (with and w/o the BioS) just to verify that I didn't screw up at some point / missed something. But I totally agree that it's more challenging (read: interesting) without the chip.
